### PR TITLE
ui build: run make watch through webpack-dashboard

### PR DESCRIPTION
**Pull request by couchand**
This change updates `make run` to pipe the webpack status through webpack-dashboard, a tool that shows the current build progress and presents a few useful stats about asset and bundle size, in addition to showing the debug output of webpack.

screen shot 2017-11-02 at 4 02 50 pm

### CODE REVIEW

**Comment by mrtracy** [comment]
I think this is fine, looks like an easy usability win; that dashboard makes much more sense to me than whatever the current spew is.  Off the top of my head, I don't think anyone is using `make watch` in a way that would conflict with this; if @benesch doesn't have an issue with it, it's LGTM.

---

Review status: 0 of 4 files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/19758#-:-Kxya1piTnLu1Y_WH-gZ:beij7w5)*

**Review by benesch — APPROVED** [review]
LGTM @couchand, modulo one safety harness.

File: pkg/ui/Makefile
Comment (benesch) [inline pkg/ui/Makefile]: Can you also wrap webpack-dashboard in $(NODE_RUN), and then stick that in a variable named `WEBPACK_DASHBOARD` at the top of the file? I know we don't currently call `make watch` from CI, but I never ever ever want to have to debug the issue detailed in `node-run` again.

**Review by benesch — APPROVED** [review]

File: pkg/ui/Makefile
Comment (couchand) [reply pkg/ui/Makefile]: Didn't do that intentionally.  Looks like for that to work properly we'd need to address your TODO regarding isatty.  I thought that would be more work that is worth it here.  Also, I think it's highly unlikely that the make watch task would be run within docker.

Comment (benesch) [reply pkg/ui/Makefile]: Ah, ok, fair enough. LGTM.

**Pull request merged**
